src/widgets/dock_area/show/leaf.rs

```diff
@@ -40,6 +40,11 @@ impl<'tree, Tab> DockArea<'tree, Tab> {
         ui.spacing_mut().item_spacing = Vec2::ZERO;
         ui.set_clip_rect(rect);
 
+        // Delay hover position one frame. On touch screens hover_pos() is None when any_released()
```

Comment:
I think it would be better to move this somewhere in the beginning of `show_inside`, as there are other parts of the widget that read `hover_pos()`.